roles/datadog_config/tasks/main.yml

```diff
@@ -0,0 +1,150 @@
+---
+
+- name: Enable RedisDB monitoring
+  become: yes
+  template: 
+    src: redis.yml.j2 
+    dest: /etc/datadog-agent/conf.d/redisdb.d/conf.yaml
+    force: yes
+
+- name: Create celery conf dir
+  become: yes
+  file:
+    path: /etc/datadog-agent/conf.d/celery.d
+    state: directory
+    mode: '0755'
+
+- name: Enable Celery monitoring
+  become: yes
+  copy: 
+    src: celery.yml 
+    dest: /etc/datadog-agent/conf.d/celery.d/conf.yaml
+    force: yes
+
+- name: Enable Tomcat monitoring
+  become: yes
+  copy: 
+    src: tomcat.yml 
+    dest: /etc/datadog-agent/conf.d/tomcat.d/conf.yaml
+    force: yes
+
+- name: Enable access to tomcat logs
+  become: yes
+  file:
+    path: /var/log/tomcat
+    state: directory
+    mode: '0750'
+    group: dd-agent
+    recurse: yes
+
+- name: Enable mod_status config in Apache
+  become: yes
+  copy: 
+    src: apache_mod_status.conf 
+    dest: /etc/httpd/conf.d/server-status.conf
+    force: yes
+
+- name: Enable access to apache logs
+  become: yes
+  file:
+    path: /var/log/httpd
+    state: directory
+    mode: '0750'
+    group: dd-agent
+    recurse: yes
+
+- name: Restart apache
+  become: yes
+  service:
+    name: httpd
+    state: restarted
+
+- name: Enable Apache monitoring
+  become: yes
+  copy: 
+    src: apache.yml 
+    dest: /etc/datadog-agent/conf.d/apache.d/conf.yaml
+    force: yes
+
+- name: Enable PostgreSQL logging 1
+  become: yes
+  lineinfile:
+    path: /var/lib/pgsql/9.6/data/postgresql.conf
+    regexp: ^log_statement =
+    line: log_statement = 'all'
+
+- name: Enable PostgreSQL logging 2
+  become: yes
+  lineinfile:
+    path: /var/lib/pgsql/9.6/data/postgresql.conf
+    regexp: ^log_file_mode =
+    line: log_file_mode = 0644
+
+- name: Enable PostgreSQL logging 3
+  become: yes
+  lineinfile:
+    path: /var/lib/pgsql/9.6/data/postgresql.conf
+    regexp: ^log_line_prefix =
+    line: log_line_prefix = '%m [%p] %d %a %u %h %c '
+
+- name: Enable PostgreSQL logging 4
+  become: yes
+  lineinfile:
+    path: /var/lib/pgsql/9.6/data/postgresql.conf
+    regexp: ^log_directory =
+    line: log_directory = '/var/log/pg_log/'
+
```

Comment:
You can do all lineinfiles tasks in a loop!
https://stackoverflow.com/questions/24334115/ansible-lineinfile-for-several-lines